CRM/Contribute/Tokens.php

```diff
@@ -134,17 +134,11 @@ public function checkActive(TokenProcessor $processor) {
    * @param \Civi\ActionSchedule\Event\MailingQueryEvent $e
    */
   public function alterActionScheduleQuery(MailingQueryEvent $e): void {
-    if ($e->mapping->getEntity() !== 'civicrm_contribution') {
+    if ($e->mapping->getEntity() !== $this->getExtendableTableName()) {
       return;
     }
-
-    $fields = $this->getFieldMetadata();
-    foreach (array_keys($this->getBasicTokens()) as $token) {
-      $e->query->select('e.' . $fields[$token]['name'] . ' AS ' . $this->getEntityAlias() . $token);
-    }
-    foreach (array_keys($this->getPseudoTokens()) as $token) {
-      $split = explode(':', $token);
-      $e->query->select('e.' . $fields[$split[0]]['name'] . ' AS ' . $this->getEntityAlias() . $split[0]);
+    foreach ($this->getReturnFields() as $token) {
```

Comment:
@eileenmcnaughton what has happened to the pseudo tokens?

Reply:
@seamuslee001 yeah - it turned out I never needed them - (or perhaps I did before I got it sane) what I was doing was adding in the field they need - ie contribution_status_id - but since that field is ALSO in getReturnFields 'under it's own name' - I don't need the spy-version

Reply:
(the tests specifically cover these )